### ISSUE TITLE
Include projections from d3-geo-projection

### ISSUE THREAD

**Issue by mattijn** [vega/vega-lite#5439]
I would like to use one of the projections as defined in the d3-geo-projections. But these are not accepted by the schema.

![image](https://user-images.githubusercontent.com/5186265/66207435-1194a700-e6b3-11e9-9b76-e00436b66503.png)

[editor-url](https://vega.github.io/editor/#/url/vega-lite/N4KABGBEAkDODGALApgWwIaQFxUQFzwAdYsB6UgN2QHN0A6agSz0QFcAjOxge1IRQyUa6ALQAbZskoAWOgAZ5dAFaxuAO0gAacLmSNq+bGGkAOOdoiQA7owAmLIwDZp5nZEIAnbkuTw8PDRxQCEt4ZDU8ZA8jAG1zMABWACYAXQsQ93QPdDExZDEjZPTLT19GWACjBQBGYqgvPHRI2JEk+Lk0nUsEHOQjAHY5VxCoPGy1WDEmvpwYgGY2zTAkl06RyDwAT0IZqHZ1NT6dAF90yCnNqNiusGBIWybMIMhYQhQPXbHWZFOoDA8ANZGO5bHZGSDUZDcWCIdBgpaQABmjFy4LU6j6x1ON2CIygD0awIh2X88FYeSJsEihFi1XidJSWLqln+QOeoN2EKhMLhfQRVK8AM51Gymy0UAF3CFAHU7A4cAoEscbtiRrj1gSnrcbutWB4CjhIPgiCRyFRaAxmGxODwhLQRJrYMg8LBSJrSFZuPrbCJqnTUMpVBpmRlEV6MHgiYjkE09Zz4NxWBEPIxkLBxRttpy8NxCN4g5BlXiwKq8ZBWUSOeDIdDYfCoMjUYaJAY8CL0GL+WMpcLRYWVToUhAQMcgA)

Are there any suggested approaches for this? Can I add all projection types of d3-geo-projection to Vega-lite?

The properties from d3-geo-projection are supported already ([doc](https://vega.github.io/vega-lite/docs/projection.html#properties)):

> In addition to the shared properties above, the following properties are supported for specific projection types in the d3-geo-projection library: coefficient, distance, fraction, lobes, parallel, radius, ratio, spacing, tilt.

Which are sub-optimal useful if the d3-geo-projections are not supported

**Comment by domoritz** [comment]
yeah, let’s add them. We could add some logic to warn users if the projections are not installed. 

Do you want to send a PR?

**Comment by mattijn** [comment]
I tested these projections and the following were working in the editor:

airy
aitoff
albers
armadillo
august
baker
berghaus
bertin1953
boggs
bonne
bottomley
bromley
chamberlinAfrica
collignon
craig
craster
eckert1
eckert2
eckert3
eckert4
eckert5
eckert6
eisenlohr
equirectangular
fahey
foucaut
gilbert
gingery
ginzburg4
ginzburg5
ginzburg6
ginzburg8
ginzburg9
gnomonic
gringorten
guyou
hammer
hammerRetroazimuthal
healpix
hill
homolosine
kavrayskiy7
lagrange
larrivee
laskowski
littrow
loximuthal
mercator
miller
modifiedStereographicAlaska
modifiedStereographicGs48
modifiedStereographicGs50
modifiedStereographicMiller
modifiedStereographicLee
mollweide
mtFlatPolarParabolic
mtFlatPolarQuartic
mtFlatPolarSinusoidal
naturalEarth2
nellHammer
orthographic
patterson
polyconic
rectangularPolyconic
robinson
satellite
sinusoidal
stereographic
times
transverseMercator
twoPointAzimuthalUsa
twoPointEquidistantUsa
vanDerGrinten
vanDerGrinten2
vanDerGrinten3
vanDerGrinten4
wagner4
wagner6
wagner7
wiechel
winkel3
interruptedHomolosine
interruptedSinusoidal
interruptedBoggs
interruptedSinuMollweide
interruptedMollweide
interruptedMollweideHemispheres
polyhedralButterfly
polyhedralCollignon
polyhedralWaterman
gringortenQuincuncial
peirceQuincuncial

I excluded the ones that didn't work. And I don't know why `naturalEarth1` does not work as it is part of the d3-geo repo ([editor](https://vega.github.io/editor/#/url/vega-lite/N4KABGBEAkDODGALApgWwIaQFxUQFzwAdYsB6UgN2QHN0A6agSz0QFcAjOxge1IRQyUa6ALQAbZskoAWOgAZ5dAFaxuAO0gAacLmSNq+bGGkAOOdoiQA7owAmLI9LnmdkQgCduS5PDw8NOMCQeACehMhGkGroeKzu6GIAoujuLACMkAC+FlBi6CHI7kYA2joQQbYxmIGQsIQo7hE4eO6syNlQGO4A1kZBoeGR1MjcsIjog5pQAGaMYmKRauoRmdllYKAQW1CVeNUbkNTxfvCsYk0HsHjIhCVp5mD3ALqrOduQXb01AxeHI2MTCJTWotbjdX5HfJaKBXTzggDqdgcOAUAFZMus1ttNtsdlU+ut3nEFjhIPgiCRyFRaAxmGxODwhLQRLt0LBkHhYKRWaQrNx3GJbCI0vdUMpVBo3rjINN+Rg8H0ZcgYnFfvBuKw1C1GMhYNDgmFfnhuIQvBKsoSwFjpZ9FT8hv9xpMZnMSbl9PhISF9bCwRD4t6Mdsg2AniBMkA)). 

If you want so, where should I add these?
